source/stylesheets/components/_panel.scss

```diff
@@ -253,17 +253,18 @@
 
 .Panel.Panel--easymoneyMapaMundi {
   position: relative;
+  z-index: 1;
 
-  height: 0;
```

Comment:
Did you check if this change didn't affect negatively other places where this component is being used?